### PR TITLE
Bug 1392059, removed obscure reference that HostPath is for testing only

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1392059

### CODE REVIEW

File: architecture/additional_concepts/storage.adoc [hunk 1]
```diff
@@ -152,12 +152,11 @@ status of the volume.
 
 === Types of Persistent Volumes
 
-{product-title} supports the following `*PersistentVolume*` plug-ins:
+{product-title} supports the following `PersistentVolume` plug-ins:
 
 ifdef::openshift-enterprise,openshift-origin[]
 - xref:../../install_config/persistent_storage/persistent_storage_nfs.adoc#install-config-persistent-storage-persistent-storage-nfs[NFS]
 - xref:../../rest_api/kubernetes_v1.adoc#v1-hostpathvolumesource[HostPath]
-(single node testing only)
```
Comment (ahardin-rh) [inline architecture/additional_concepts/storage.adoc]: @markturansky Do you think it is okay to remove the line here about testing?

Comment (ahardin-rh) [inline architecture/additional_concepts/storage.adoc]: @markturansky Bump. Pinging you since it looks like you introduced this line into the docs. Thanks!

Comment (markturansky) [inline architecture/additional_concepts/storage.adoc]: Well, it was originally intended for single node testing but I think PV HostPath has evolved into a thing of its own.

Comment (ahardin-rh) [inline architecture/additional_concepts/storage.adoc]: @markturansky Okay, thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[Additional Concepts -> Persistent Storage]
|Removed statement indicating that HostPath is for testing only.
%

**Pull request merged**